### DISCUSSION TITLE
Dev refact some features need to be more modular

### DISCUSSION THREAD

**Discussion by Pipazoul** [datatlas-erasme/front#112] [Ideas]
While working on canographia I've realised that there where many features the needed to reworked to be modular
**Features** : 
- POI Sidepanel Display is hardcoded for now => parse the kepler data based on the fieldtodisplay var
- Mapping system to rewrite filters names into human readable names ex : status_projet => "status projets en cours"
- To be able to choose for each layer if its points should be displayed on the sidepanel or only using a popover when clicked
- Layer order when displayed in the sidepanel

@aangelot  should thoses features be added in the issues ?

**Comment by aangelot** [comment]
Hi Yassin ! 

Yes they must be added, especially the two first you raised are very important ! 
The third one can be linked to the different map styles. But indeed it also can be added as an issue for more modulariy. 
The fourth one will quickly be not optionnal. 

Do you add them ?

**Reply by Pipazoul** [reply]
Referenced as issues 
#117 
#118
#119
